core-header-panel.html

```diff
@@ -298,32 +287,19 @@
 
         _atTop: {
           value: true
-        },
-
-        _dropShadowClass: {
-          computed: 'computeDropShadowClass(_atTop, mode, shadow)'
-        },
-
-        _mainContainerClass: {
-          computed: 'computeMainContainerClass(mode)'
         }
-
       },
 
-      computeDropShadowClass: function(_atTop, mode, shadow) {
+      _computeDropShadowHidden: function(_atTop, mode, shadow) {
         var needsShadow = _atTop &&
           MODE_CONFIGS.shadowMode[this.mode] ||
           MODE_CONFIGS.noShadow[this.mode];
```

Comment:
Totally not in the diff, but this whole conditional could live in an accessor to save us creating a temp.